gen/nvml/init.go

```diff
@@ -24,7 +24,7 @@ import "C"
 
 const (
 	nvmlLibraryName      = "libnvidia-ml.so.1"
-	nvmlLibraryLoadFlags = dl.RTLD_LAZY | dl.RTLD_GLOBAL
+	nvmlLibraryLoadFlags = dl.RtldLazy | dl.RtldGlobal
```

Comment:
Update: OK, I see that the change was made here. See my earlier comment on adding something like `dl.DefaultRtldFlags` here.